addons/mysql-exporter/README.md

```diff
@@ -0,0 +1,11 @@
+
+# Mysql Exporter
+
+Prometheus exporter for MySQL server metrics.
+
+Supported versions:
+
+MySQL >= 5.6.
+MariaDB >= 10.3
+
```

Comment:
Please give a example application tell user how to use this addon.

Reply:
Already added.